### PR TITLE
AUDIO: Set up new player

### PR DESCRIPTION
**Pull request by ajwl**
## What does this change?

Basic CSS changes to the audio player, to bring it into line with the new design. 

Missing:
- functionality check
- new style progress bar 
- volume clearup 
- breakpoints > mobile. 

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by tomrf1 — APPROVED** [review]
looks good

File: static/src/javascripts/projects/common/modules/audio/AudioPlayer.js [hunk 1]
```diff
@@ -5,383 +5,331 @@ import {
...
+            muted: false,
+            scrubbing: false,
             currentTime: 0,
-            duration: NaN,
-            volume: NaN,
-            bins: null,
-            interval: NaN,
             currentOffset: 0,
+            currentOffsetPx: 0,
+            duration: 3000,
```
Comment (ajwl) [inline static/src/javascripts/projects/common/modules/audio/AudioPlayer.js]: Should probably initialise this at 0? (think i added this to test something out)

File: static/src/javascripts/projects/common/modules/audio/AudioPlayer.js [hunk 2]
```diff
@@ -5,383 +5,331 @@ import {
...
+import volumeOn from 'svgs/journalism/audio-player/volume-on.svg'; // eslint-disable-line
+import volumeOff from 'svgs/journalism/audio-player/volume-on.svg'; // eslint-disable-line
 import fastBackward from 'svgs/journalism/audio-player/fast-backward.svg';
 import fastForward from 'svgs/journalism/audio-player/fast-forward.svg';
 import fastBackwardActive from 'svgs/journalism/audio-player/fast-backward-active.svg';
 import fastForwardActive from 'svgs/journalism/audio-player/fast-forward-active.svg';
 
 import waveW from 'svgs/journalism/audio-player/wave-wide.svg';
-import { formatTime, sendToOphan, checkForTimeEvents } from './utils';
+import { sendToOphan, checkForTimeEvents } from './utils';
```
Comment (ajwl) [inline static/src/javascripts/projects/common/modules/audio/AudioPlayer.js]: Meant to mention this before. But the `checkForTimeEvents` function is too restrictive and should be replaced with `monitorPercentPlayed`. Should also move into utils for consistency https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/audio/inArticleDataEvents.js

**Review by ajwl — COMMENTED** [review]
A few small changes.

**Comment by regiskuckaertz** [comment]
Note for later. We can merge the play and pause buttons, showing/hiding the correct SVG path depending on.

**Pull request merged**